specification/components/stu3/schemas/endpoints/a010-patient-service-search.yaml

```diff
@@ -24,6 +24,8 @@ description: |
   ## Related endpoints
   Once you have identified appropriate services for the patient you can create a referral by choosing a shortlist of bookable and/or triage services using [[HYPERLINK_A011]]. You may then subsequently be able to use [[HYPERLINK_A016]] to book an appointment to a directly bookable service if an appointment slot is available, defer the booking responsibility to a directly bookable service if no appointment slot is available, or refer a patient into a triage service.
 
+  If using this endpoint for [[HYPERLINK_A031]], you will need to use the Rule Context Organisation parameter. This will ensure that the list of services that are returned for the given referral are valid.
```

Comment:
This might be a mistranslation on my side, so please ignore if so, but as we are talking about the "list" being valid (as a list could contain just one service), shouldn't this sentence be "This will ensure that the list of services that are returned for the given referral is valid."?

Reply:
Hmm I've just taken this line from the story but I think this is grammatically correct (though I have to admit my gramma isn't always fantastic). I believe this is correct as it is indicating that each service on the list is valid rather than indicating that the entire list is valid. I think this could also be written as
"This will ensure that the list of services returned is valid for the Referral" or "This will ensure that the services returned are valid for the Referral" which might be a bit more readable. I'll mention it to the BAs to see if we can reword this sentence a bit to make it easier to read 🙂.

Reply:
The updated string makes more sense to me :) Thanks for checking it!